MagazineLayout/Public/MagazineLayout.swift

```diff
@@ -766,6 +796,12 @@ public final class MagazineLayout: UICollectionViewLayout {
   private var footerLayoutAttributes = [ElementLocation: MagazineLayoutCollectionViewLayoutAttributes]()
   private var backgroundLayoutAttributes = [ElementLocation: MagazineLayoutCollectionViewLayoutAttributes]()
 
+  // These properties are used to keep the layout attributes copies used for insert/delete
+  // animations up-to-date as items are self-sized. If we don't keep these copies up-to-date, then
+  // animations will start from the estimated height.
+  private var itemLayoutAttributesForInFlightAnimations = [IndexPath: UICollectionViewLayoutAttributes]()
```

Comment:
Doesn't feel like `InFlight` is the right term here:

1. Not a UIKit/Cocoa term of art.
2. I don't think the animations are actually already in progress. If they were, updating the attributes wouldn't do anything. We could calling them `pendingAnimations`?

Reply:
Yeah good point, I like pending